packages/editor/src/store/selectors.js

```diff
@@ -572,7 +572,7 @@ export function isEditedPostDateFloating( state ) {
 	const date = getEditedPostAttribute( state, 'date' );
 	const modified = getEditedPostAttribute( state, 'modified' );
 	const status = getEditedPostAttribute( state, 'status' );
-	if ( status === 'draft' || status === 'auto-draft' ) {
+	if ( status === 'draft' || status === 'auto-draft' || status === 'pending' ) {
```

Comment:
I think the change makes sense! Would it be possible to add a test case covering pending state to the isEditedPostDateFloating selector test suite?

Reply:
@jorgefilipecosta Took care of this here https://github.com/WordPress/gutenberg/pull/13256